gnucash/gnome-utils/gnc-tree-view-split-reg.c

```diff
@@ -2926,7 +2926,7 @@ gtv_sr_titles (GncTreeViewSplitReg *view, RowDepth depth)
             {
             default: //FIXME These if statements may not be required
                 if (depth == TRANS1 || depth == TRANS2 || depth == SPLIT3)
-                    gtk_tree_view_column_set_title (tvc, _("Rate"));
+                    gtk_tree_view_column_set_title (tvc, C_("In register", "Rate"));
```

Comment:
This column is hidden so its heading shouldn't be translated.

Reply:
Then it shouldn't have be `_("Rate")` either? 

If I understand how gettext works correctly, in the original code, this heading is also got translated to the string inside .po files.

Reply:
> Then it shouldn't be _("Rate") neither?

Right, just `"Rate"`, maybe with a comment noting that it's a hidden column.